tests/integration/cassandra_auth_test.go

```diff
@@ -89,67 +101,75 @@ func (s *CassandraSSLTestSuite) TestCassandraIntegration_SSL() {
 }
 
 func (s *CassandraSSLTestSuite) TestCassandraIntegration_WrongConfig() {
-	for _, Hostname := range testutils.Hostnames {
-		t := s.T()
+	t := s.T()
+
+	testName := t.Name()
 
-		testName := t.Name()
+	testCases := []struct {
+		name          string
+		config        map[string]string
+		expectedError string
+	}{}
 
-		testCases := []struct {
+	for _, cassandraConfig := range testutils.CassandraConfigs {
+		testCases = append(testCases, struct {
 			name          string
 			config        map[string]string
 			expectedError string
 		}{
-			{
-				name: "WrongPassword",
-				config: map[string]string{
-					"METRICS": "true",
-					"TIMEOUT": SSLConnectionTimeout,
-
-					"HOSTNAME": Hostname,
-					"USERNAME": "wrong",
-					"PASSWORD": "wrong",
-
-					"TRUST_STORE":          testutils.TruststoreFile,
-					"TRUST_STORE_PASSWORD": testutils.TruststorePassword,
-					"KEY_STORE":            testutils.KeystoreFile,
-					"KEY_STORE_PASSWORD":   testutils.KeystorePassword,
-
-					"NRIA_CACHE_PATH": fmt.Sprintf("/tmp/%v.json", testName),
-				},
-				expectedError: "Authentication failed! Invalid username or password",
+			name: "WrongPassword",
```

Comment:
I'd add some additional details to the name (it would make it easier to identify if anything starts failing). For instance:

```go
name : fmt.Sprintf("WrongPasword - %s", cassandraConfigVersion)
```

Same for the other test cases

Reply:
I have separated the logic of the tests and also added version numbers for testnames